pkg/controller/controller_test.go

```diff
@@ -2470,6 +2481,7 @@ func assertServiceBindingOperationInProgressWithParameters(t *testing.T, obj run
 		assertServiceBindingInProgressPropertiesNil(t, obj)
 	}
 	assertServiceBindingExternalPropertiesUnchanged(t, obj, originalBinding)
+	assertServiceBindingUnbindStatus(t, obj, originalBinding.Status.UnbindStatus)
```

Comment:
The `UnbindStatus` should be Required. If there is an operation is progress, then a Bind request has been sent to the broker and an Unbind request has not yet been successful. Hence, an unbind is required.

Reply:
Fixed.